tests/utils/test_file.py

```diff
@@ -128,10 +129,9 @@ def test_find_path_from_directory_glob_ignore(self):
         ]
         should_not_ignore = [
             "test_on_kill.py",
-            "test_dont_ignore_this.py",
+            # "test_dont_ignore_this.py",
```

Comment:
I cannot figure out why this is in the 'should ignore' list. The glob pattern actually excludes this. So if someone can shed some light on this that would be appreciated.